spec/unit/lib/cloud_controller/opi/instances_client_spec.rb

```diff
@@ -12,7 +12,12 @@
     )
   end
 
-  let(:process) { double(guid: 'my-process-guid', version: 'my-version-guid') }
+  let(:process) { VCAP::CloudController::ProcessModel.make(
```

Comment:
nit: this indentation is a little strange

Reply:
i blame rubocop